apps/wso2-digital-wallet/src/pages/Home/Home.css

```
@@ -14,7 +14,10 @@
   position: relative;
   display: flex;
   flex-direction: column;
-  height: calc(100vh - 68px - var(--safe-area-bottom, 8px));
+  height: calc(
+    100vh - 68px - var(--safe-area-top, 0px) - var(--topbar-height, 44px) -
```

Stylelint (17.11.1)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)